src/traversing.js

```diff
@@ -168,10 +159,18 @@ jQuery.each({
 		return elem.contentDocument || jQuery.merge( [], elem.childNodes );
 	}
 }, function( name, fn ) {
+	var // Sorting (which implicitly de-dupes) is required for non-adjacent traversals
+		sort = name !== "prev" && name !== "next",
```

Comment:
Sorting makes a _huge_ difference, even on relatively small input sets: http://jsperf.com/jquery-fn-next-sorting

To be honest, I'm upset about removing this exemption, and it may actually be worthwhile to squeeze something back in (although unfortunately, the operations required to detect a need for sorting are basically identical to those required to perform a sort). But I think that `$largeCollection.children()` is probably more rare than `$largeCollection.next()`, and would like to leave this PR as-is.